app/js/model/list_model.js

```diff
@@ -6,6 +6,7 @@ export default class ListModel extends Model {
       'http://fxos.github.io/sharing/dist/app/manifest.webapp': {
         manifestURL: 'http://fxos.github.io/sharing/dist/app/manifest.webapp',
         name: 'sharing',
+        icon: 'https://raw.githubusercontent.com/fxos/sharing/master/app/icons/Icon_P2P_Sharing.png',
```

Comment:
Maybe instead of duplicating all of this information, we could pull down the manifest and get it from there.